credsweeper/utils/util.py

```diff
@@ -285,24 +306,16 @@ def get_xml_data(file_path: str) -> Tuple[Optional[List[str]], Optional[List[int
             List of formatted string(f"{root.tag} : {root.text}")
 
         """
-        lines = []
-        line_nums = []
         try:
             with open(file_path, "r") as f:
                 xml_lines = f.readlines()
-            tree = etree.fromstringlist(xml_lines)
-            for element in tree.iter():
-                tag = Util._extract_element_data(element, "tag")
-                text = Util._extract_element_data(element, "text")
-                lines.append(f"{tag} : {text}")
-                line_nums.append(element.sourceline)
+            return Util.get_xml_from_lines(xml_lines)
         except Exception as exc:
             logger.error(f"Cannot parse '{file_path}' to xml {exc}")
             return None, None
-        return lines, line_nums
 
     @staticmethod
-    def _extract_element_data(element, attr) -> str:
+    def extract_element_data(element, attr) -> str:
```

Comment:
I think `extract_element_data()` should change with `_extract_element_data()`, it's only used in `Util` class.